### PR TITLE
Twig Extensions Reference minor brush-up

### PR DESCRIPTION
**Pull request by mpdude**
Especially clarify that the "Twig Extensions Repository" has nothing to do with Symfony Standard Edition

### CODE REVIEW

File: reference/twig_reference.rst [hunk 1]
```diff
@@ -7,17 +7,33 @@ Symfony Twig Extensions
...
+and :ref:`tests <reference-twig-tests>` to seamlessly integrate the various
+Symfony Components with Twig templates. The following sections describe these 
+extra features.
 
-There may also be tags in bundles you use that aren't listed here.
+.. tip::
+
+    Technically, most of the extensions live in the `Twig Bridge`_. That code
+    might give you some ideas when you need to write your own Twig extension
+    as described in :doc:`/templating/twig_extension.rst`.
```
Comment (xabbuh) [inline reference/twig_reference.rst]: You have to omit the file extension here.

Comment (mpdude) [inline reference/twig_reference.rst]: Thanks, fixed

File: reference/twig_reference.rst [hunk 2]
```diff
@@ -7,17 +7,33 @@ Symfony Twig Extensions
...
-from the `Twig Reference`_).
+a lot of built-in functions, filters, tags and tests. You can learn more about
+them from the `Twig Reference`_.
 
-Symfony adds custom extensions on top of Twig to integrate some components
-into the Twig templates. The following sections describe the custom
-:ref:`functions <reference-twig-functions>`, :ref:`filters <reference-twig-filters>`,
-:ref:`tags <reference-twig-tags>` and :ref:`tests <reference-twig-tests>`
-that are available when using the Symfony Core Framework.
+The Symfony Framework adds quite a few extra :ref:`functions <reference-twig-functions>`,
```
Comment (xabbuh) [inline reference/twig_reference.rst]: framework

File: reference/twig_reference.rst [hunk 3]
```diff
@@ -7,17 +7,33 @@ Symfony Twig Extensions
...
+:ref:`filters <reference-twig-filters>`, :ref:`tags <reference-twig-tags>`
+and :ref:`tests <reference-twig-tests>` to seamlessly integrate the various
+Symfony Components with Twig templates. The following sections describe these
```
Comment (xabbuh) [inline reference/twig_reference.rst]: components

File: reference/twig_reference.rst [hunk 4]
```diff
@@ -7,17 +7,33 @@ Symfony Twig Extensions
...
+.. tip::
+
+    Technically, most of the extensions live in the `Twig Bridge`_. That code
+    might give you some ideas when you need to write your own Twig extension
+    as described in :doc:`/templating/twig_extension`.
+    
+.. note::
 
+    This reference only covers the Twig extensions provided by the Symfony 
+    Framework. You are probably using some other bundles as well, and those
```
Comment (xabbuh) [inline reference/twig_reference.rst]: framework

File: reference/twig_reference.rst [hunk 5]
```diff
@@ -721,14 +737,8 @@ The available attributes are:
...
-Those bundles can have other Twig extensions:
-
-* **Twig Extensions** includes some interesting extensions that do not belong
-  to the Twig core. You can read more in `the official Twig Extensions documentation`_.
-
 .. _`Twig Reference`: http://twig.sensiolabs.org/documentation#reference
-.. _`the official Twig Extensions documentation`: http://twig.sensiolabs.org/doc/extensions/index.html
+.. _`Twig Extensions repository`: https://github.com/twigphp/Twig-extensions
+.. _`Twig Extensions documentation`: http://twig-extensions.readthedocs.io/en/latest/
+.. _`KernelEvents`: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/KernelEvents.php
```
Comment (xabbuh) [inline reference/twig_reference.rst]: This isn't used anywhere here, is it?

**Comment by mpdude** [comment]
@xabbuh Thanks for the review, fixed all your remarks

**Comment by xabbuh** [comment]
👍 

Status: Reviewed

**Review by HeahDude — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@mpdude I'm really sorry but your PR needs to be rebased. Could you do that for us? Thanks!

**Comment by mpdude** [comment]
@javiereguiluz done, merged master

**Comment by javiereguiluz** [comment]
a@mpdude thanks for your contribution and for the rebase! It's merged now.

**Pull request closed without merging**